hack/dockerfiles/govulncheck.Dockerfile

```diff
@@ -0,0 +1,25 @@
+# syntax=docker/dockerfile:1
+
+ARG GO_VERSION=1.21.12
```

Comment:
OH! This should be 1.21.13 now; https://github.com/moby/moby/commit/b24c2e95e57c5927765fa1ff5820576dda6e6242